app/views/games/_action_buttons.html.erb

```diff
@@ -0,0 +1,28 @@
+<div class="btn-group">
+  <% if game.state_created? %>
+    <% if game.game_players.count >= Game::MIN_PLAYERS_PER_GAME %>
+      <%= button_to start_game_game_path(game), method: :put, class:"text-decoration-none btn btn-empty btn-sm", style: "padding: 0.1rem !important;" do %>
+        <i class="bi bi-play-fill" style="font-size: 1.1rem; color: #0d6efd;"></i>&nbsp;
+      <% end %>
+    <% end %>
+    <%= button_to cancel_game_game_path(game), method: :put, class:"text-decoration-none btn btn-empty btn-sm", style: "padding: 0.1rem !important;" do %>
+      <i class="bi bi-x-circle-fill" style="font-size: 1.1rem; color: #0d6efd;"></i>&nbsp;
+    <% end %>
+  <% elsif game.state_in_progress?%>
+    <%= button_to cancel_game_game_path(game), method: :put, class:"text-decoration-none btn btn-empty btn-sm", style: "padding: 0.1rem !important;" do %>
+      <i class="bi bi-x-circle-fill" style="font-size: 1.1rem; color: #0d6efd;"></i>&nbsp;
+    <% end %>
+    <%= button_to complete_game_game_path(game), method: :put, class:"text-decoration-none btn btn-empty btn-sm", style: "padding: 0.1rem !important;" do %>
+      <i class="bi bi-check-circle-fill" style="font-size: 1.1rem; color: #0d6efd;"></i>&nbsp;
+    <% end %>
+  <% end %>
```

Comment:
I feel like you have to many if/else/if/else here. You need 3 cases
1. add start_game if it is created and the players count is between 2 and 8
2. add cancel_game if it is created or in progress
3. add completed_game if it is in progress

Reply:
you should also use something like `update_game_path(game, state: desired_state)`

Reply:
Fixed in my commit